aspnetcore/migration/inc/samples/remote-authentication/AspNetCore.cs

```diff
@@ -0,0 +1,19 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// <snippet_AddSystemWebAdapters>
+builder.Services.AddSystemWebAdapters()
+    .AddRemoteAppClient(options =>
+    {
+        options.RemoteAppUrl = new Uri(builder.Configuration["ReverseProxy:Clusters:fallbackCluster:Destinations:fallbackApp:Address"]);
+        options.ApiKey = builder.Configuration["RemoteAppApiKey"];
```

Comment:
This is different because the original snippet (`builder.Configuration("RemoteAppApiKey")`) contains a compile error.

Reply:
> This is different because the original snippet (`builder.Configuration("RemoteAppApiKey")`) contains a compile error.

That's a big reason we vastly prefer to import snippets from project files.